packages/request-manager/src/controller.ts

```diff
@@ -21,6 +21,50 @@ export class TorController extends EventEmitter {
         this.controlPort = new TorControlPort(options, this.onMessageReceived.bind(this));
     }
 
+    getTorConfiguration(
```

Comment:
If I understand it correctly, `TorController` already knows `port`, `controlPort` and `authFilePath` when created. Wouldn't it make sense to use these values for the produced configuration and pass only `processId` as a parameter?

Reply:
You are totally right, I just made changes. Thank you for catching that!